k8s/base/redis/deployment.yaml

```
@@ -0,0 +1,78 @@
+      labels:
+        app: redis
+        component: cache
+    spec:
+      containers:
+      - name: redis
+        image: redis:7-alpine
+        ports:
+        - containerPort: 6379
+          name: redis
+        command:
+          - redis-server
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)